.vale/styles/RedHat/VisualLanguage.yml

```diff
@@ -0,0 +1,22 @@
+---
+extends: existence
+ignorecase: true
+level: suggestion
+link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/visuallanguage/
+message: "It is acceptable to use visual styling in documentation, but do not rely solely on visual aspects to convey meaning."
+tokens:
+  - above
+  - below
+  - right
+  - left
```

Comment:
Can we rework this list to include appropriate quickfix suggestions as outlined in the SSG for directional terms?

e.g., 
```
above: previous|preceding
below: following
```

The rule would then need to be a https://vale.sh/docs/topics/styles/#substitution rule. This may not be suitable for all these terms though.

Reply:
I thought about that, but I think that in a lot of cases the rewriter will be more complicated than switching the word out. For example, if we're talking about `the menu on the left side of the page`, which I see quite often, substituting left for something won't work. The best practice is to name the menu or ui control. And getting rid of left likely means rewriting the sentence.

It's also allowable to use visual language if you also provide a non visual way to find it.

What do you think?

Reply:
Looking at https://www.ibm.com/docs/en/ibm-style?topic=word-usage#section-use-with-caution-left-adjective, the rule as you have it here would flag left even when it was used correctly, "The list of tables is displayed in the Table List pane on the left side of the window".

I'm not sure how we could reduce false positives in this case with simple regex. I wonder could a sequence rule be useful? https://vale.sh/docs/topics/styles/#sequence

Reply:
I think you're right. I'm struggling with the sequence rule because the pattern that comes before or after could be anything. 

I'm not sure there's a way to reduce false positives in this case. And I was aware of that when I started this rule. It is important. What if we made it a warning, for example, if you're going to use these visual words, make sure you have it another way too?

Reply:
Not quite what you are looking for, but here are swap terms (bad:good) lifted from the ISG: 

```
bottom left: lower left
bottom right: lower right
left most: leftmost
left-hand: left
left-justified: left-aligned
left-justify: left-align
left-most: leftmost
right double-click: double right-click
right-hand: right
right-justified: right-aligned
right-justify: right-align
right-most: rightmost
```

Reply:
Another thought - grep search across a few repos, see if there are common misused terms we can flag.

Reply:
more suggestions:

below: earlier|later|following
above: later|previous|preceding